docs/sdkx_ios/delegates.mdx

```diff
@@ -265,19 +265,59 @@ func handleHelpshiftEvent(_ eventName: String, withData data: [AnyHashable : Any
 
 ### Agent Message Received Event
 
-This event is triggered when the user receives any message from an agent in a conversation. It might be a text message, bot message, or a message triggered via automation.
+This event is triggered when the user receives any message from an agent in a conversation. It might be a text message or an attachment. This delegate is not triggered for bot messages or messages sent via automations.
```

Comment:
The event is also called when there's an App Review Request sent by the dashboard.

Reply:
@arif-helpshift That constitutes as agent sent message right?

Reply:
@jaydeep-helpshift We've 4 'type'(s) - 1. text, 2. attachment, 3. app_review_request, 4. screenshot_request.